pkg/sdk/testint/streamlits_integration_test.go

```diff
@@ -0,0 +1,141 @@
+package testint
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/random"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_Streamlits(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	databaseTest, schemaTest := testDb(t), testSchema(t)
+
+	cleanupStreamlitHandle := func(id sdk.SchemaObjectIdentifier) func() {
+		return func() {
+			err := client.Streamlits.Drop(ctx, sdk.NewDropStreamlitRequest(id))
```

Comment:
you can use IfExists instead of checking the error

Reply:
good idea